source/beamline/BioXAS/BioXASZebraPulseControl.cpp

```diff
@@ -200,8 +203,10 @@ void BioXASZebraPulseControl::setInputValue(int value)
 
 void BioXASZebraPulseControl::setEdgeTriggerValue(int value)
 {
-	if (!edgeTriggerControl_->withinTolerance(double(value)))
-		edgeTriggerControl_->move(double(value));
+	if (edgeTriggerValue_ != value) {
+		edgeTriggerValue_ = value;
```

Comment:
I guess it depends! I see what you mean, if someone caput's a new value to the edge trigger control or makes a value change using the EDM screen, the edgeTriggerValue_ is no longer identical to the value of the edgeTriggerControl_ as written. 

The point of storing the value is so that it can be specified before the control connects and applied to the control once it is connected, intentionally overriding any previous values on application startup. If someone changes the value outside of AM, I think it's an indication of something going wrong or that the control is being misused. I'm not sure whether that is best represented as having the edgeTriggerValue_ synchronized to the control's value or not.